bin/npg_delete_dr_runs

```diff
@@ -14,51 +14,98 @@ use npg_tracking::Schema;
 our $VERSION = '0';
 
 my $help;
-GetOptions ('help|?'   => \$help)
+my $files;
+my $folders;
+my $no_dry_run;
+
+GetOptions ('help|?'   => \$help,
+            'no-dry-run'=> \$no_dry_run,
+            'files'    => \$files,
+            'folders'  => \$folders)
   or croak("Error in command line arguments\n");
 
+if ($files and $folders) {
+    print STDERR "You can't have --files AND --folders. Choose one.\n";
+    pod2usage(1);
+}
+
 pod2usage(0) if $help;
 
 my $rs=npg_tracking::Schema->connect()->resultset(q(RunStatusDict))
     ->search({description=>[q[run cancelled],q[data discarded],q[qc complete]]})
     ->related_resultset(q(run_statuses))->search({q(run_statuses.iscurrent)=>1})
     ->related_resultset("run")->search({q(run.folder_path_glob)=>{LIKE=>q(%/esa-%)}});
 
+my $cmd = '';
 while (my $run=$rs->next()) {
     my $status = $run->current_run_status_description || "";
     my $tag = $run->is_tag_set("staging");
     my $run_folder = $run->folder_name || "";
     if ($run_folder and !$tag ) {
-        print $run_folder, "\t", $run->id_run, "\t", $run->folder_path_glob, "\t", $status, "\n";
+        my $prefix = substr($run_folder,0,4);
+        my $dname = "/exports/$prefix/$run_folder";
+        if ($files and !$folders) {
+            $cmd .= "if [ -d $dname ]\\\; then rm $dname/Data/Intensities/BaseCalls/L\*/C\*/\*.cbcl\\\; fi\\\;";
+        }
+        if ($folders and !$files) {
+            $cmd .= "if [ -d $dname ]\\\; then rm -r $dname\\\; fi\\\;";
+        }
+        if (!$files and !$folders) {
+            $cmd .= "if [ -d $dname ]\\\; then ls -ld $dname\\\; fi\\\;";
+        }
     }
 }
+chop $cmd;
+if ($no_dry_run) {
+    system("echo $cmd | ssh ukbb\@seq-stor-01 bash");
```

Comment:
We could open filehandles for each `ssh` process forked and write commands run folder by runfolder to each filehandle. Printing to those avoids the `echo` and extra level of escaping. 
I fear `echo $cmd` might grow too large for the shell.
What about running `bash -e` as we're not expecting failures given the testing for directories.

Reply:
There is more to closing filehandles like this than is currently in the code, see https://perldoc.perl.org/functions/close.html